server/src/test/java/io/crate/integrationtests/LogicalReplicationITest.java

```diff
@@ -283,6 +285,9 @@ public void test_subscribing_to_single_publication() throws Exception {
 
         executeOnPublisher("CREATE PUBLICATION pub1 FOR TABLE doc.t1");
 
+        executeOnPublisher("CREATE USER " + SUBSCRIBING_USER);
+        executeOnPublisher("GRANT DQL ON TABLE doc.t1 TO " + SUBSCRIBING_USER);
+
```

Comment:
I'd suggest to move that out into a helper method `createPublication(name, tables...)`.